packages/jsapi-utils/src/TableUtils.ts

```diff
@@ -1776,7 +1778,9 @@ export class TableUtils {
       );
     }
 
-    return dh.FilterValue.ofNumber(TableUtils.removeCommas(value));
+    return dh.FilterValue.ofNumber(
+      TableUtils.removeCommas(value) as unknown as number
+    );
```

Comment:
Gross. Do we have a follow up ticket to improve JS API? I feel like I commented on this at the time on the JS API review as well as `ofNumber` needing to take `string`, but unsure why that wasn't added to the Union type.

Reply:
https://github.com/deephaven/deephaven-core/issues/5082